api/proto/teleport/discoveryconfig/v1/discoveryconfig.proto

```diff
@@ -50,3 +53,30 @@ message DiscoveryConfigSpec {
   // AccessGraph is the configurations for syncing Cloud accounts into Access Graph.
   types.AccessGraphSync access_graph = 6;
 }
+
+// DiscoveryConfigStatus holds dynamic information about the discovery configuration
+// running status such as errors, state and count of the resources.
+message DiscoveryConfigStatus {
```

Comment:
Can we add "last sync time" here as well?

Reply:
Added in [e803e3b](https://github.com/gravitational/teleport/pull/40136/commits/e803e3b32fd641a44aff0571fc4d693c842d7478)